Packs/Hoxhunt/Integrations/HoxhuntV2/HoxhuntV2.py

```diff
@@ -639,7 +645,7 @@ def get_modified_remote_data_command(client: Client, args: dict, params: dict):
     remote_args = GetModifiedRemoteDataArgs(args)
     last_update = remote_args.last_update
     last_update_parsed = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
-    last_update_utc = last_update_parsed.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'  # type: ignore
```

Comment:
Let's only change what's necessary for the command you want to modify (hoxhunt-incident-soc-feedback-send) to avoid from unwanted impacting on users.